db/model/event.js

```diff
@@ -68,6 +81,25 @@ module.exports = function event(seq, dataTypes) {
         });
       },
     },
+    hooks: {
+
+      afterCreate: (instance) => {
+        const changedKeys = Object.keys(instance._changed);
+        const ignoreAttributes = ['isDeleted'];
+        return realTime.publishObject(instance.toJSON(),
+          botEventNames.add, changedKeys, ignoreAttributes, pubOpts);
+      },
+
+      afterUpdate(instance /* , opts */) {
+        return realTime.publishObject(instance.toJSON(),
+          botEventNames.upd, null, null, pubOpts);
```

Comment:
curious why not sending changed keys and ignore attributes here or in afterDelete, like you do in afterCreate?

Reply:
overlooked this; ive added the changes